pom.xml

```diff
@@ -51,6 +51,7 @@
         <io.confluent.kafka-rest.version>7.7.0-0</io.confluent.kafka-rest.version>
         <!-- Allow enough heap space for integration tests with both Kraft and Zookeeper -->
         <argLine>-Xmx4g -Xms2g</argLine>
+        <guava.version>32.0.1-jre</guava.version>
```

Comment:
@shayelkin kafka-rest use guava.version from [common](https://github.com/confluentinc/common/blob/dcd1bdb0a55c0b2c2324c595c79bb13b6b8194c2/pom.xml#L64), which is already a good version, I think this PR might not be needed.